spec/classes/foreman_proxy__plugin__ansible_spec.rb

```diff
@@ -45,16 +68,25 @@
 
         let :params do
           {
-            :enabled           => true,
-            :ansible_dir       => '/etc/ansible-test',
-            :working_dir       => '/tmp/ansible',
-            :host_key_checking => true,
-            :stdout_callback   => 'debug',
+            :enabled             => true,
+            :ansible_dir         => '/etc/ansible-test',
+            :working_dir         => '/tmp/ansible',
+            :host_key_checking   => true,
+            :stdout_callback     => 'debug',
+            :manage_runner_repo  => false
```

Comment:
Nit: could you add the trailing comma? We should enforce this via rubocop but until then ... :)

While you're changing this, mind switching it to `manage_runner_repo: false` style and removing the alignment? I've been doing this in other places in our test code because the alignment just creates noise.

Reply:
I like it much more actually :)